RecoMuon/DetLayers/src/MuonGEMDetLayerGeometryBuilder.cc

```diff
@@ -31,13 +31,13 @@ MuonGEMDetLayerGeometryBuilder::buildEndcapLayers(const GEMGeometry& geo) {
     int iendcap = (endcap==1) ? 0 : 1; // +1: forward, -1: backward
       
     for(int station = GEMDetId::minStationId; station <= GEMDetId::maxStationId; ++station) {      
-      for(int layer = GEMDetId::minLayerId+1; layer <= GEMDetId::maxLayerId; ++layer) { 
+      for(int layer = GEMDetId::minLayerId; layer <= GEMDetId::maxLayerId; ++layer) { 
 
 	vector<int> rolls, rings, chambers; 
 	for(int ring = GEMDetId::minRingId; ring <= GEMDetId::maxRingId; ++ring) rings.push_back(ring);
 	for(int chamber = GEMDetId::minChamberId+1; chamber <= GEMDetId::maxChamberId; chamber++ )
 	  chambers.push_back(chamber);
-	for(int roll = GEMDetId::minRollId+1; roll <= GEMDetId::maxRollId; ++roll)
+	for(int roll = GEMDetId::minRollId; roll <= GEMDetId::maxRollId; ++roll)
```

Comment:
yes, back then there was an odd issue with gemRechits not being found properly.
I forgot about this change...
This is needed since seeding is done with gemSegments and gemSegments are found by superChambers.